client/extensions/woocommerce/components/table/table-row/index.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import classnames from 'classnames';
+
+const Row = ( { values, className, isHeader } ) => {
+	const El = isHeader ? 'th' : 'td';
+	const rowClasses = classnames( 'table-row', className, {
+		'is-header': isHeader,
+	} );
+
+	return (
+		<tr className={ rowClasses }>
+			{ values.map( ( value, index ) => {
+				if ( index === 0 ) {
+					return (
+						<El className={ 'table-row__cell is-title-cell' } key={ value + index }>
+							<div className="table-row__cell-title" >{ value }</div>
```

Comment:
Add in a `title={ value.toString() }` property to the div so that you get the full value in a tooltip:
![screen shot 2017-06-08 at 2 02 29 pm](https://user-images.githubusercontent.com/1160732/26927711-487e0038-4c53-11e7-9a76-61422eb7465b.png)